clickhouse-native-jdbc/src/test/java/com/github/housepower/jdbc/tool/EmbeddedDriver.java

```diff
@@ -56,7 +55,7 @@ public boolean jdbcCompliant() {
     }
 
     @Override
-    public Logger getParentLogger() throws SQLFeatureNotSupportedException {
+    public java.util.logging.Logger getParentLogger() throws SQLFeatureNotSupportedException {
         return null;
```

Comment:
why not use import ?

Reply:
I add a checkstyle rule forbid import `org.slf4j` and `java.util.logging` to avoid using them directly. So we need to add exceptions for them or using complete class name when necessary.